github/git_blobs.go

```diff
@@ -38,6 +39,23 @@ func (s *GitService) GetBlob(ctx context.Context, owner string, repo string, sha
 	return blob, resp, err
 }
 
+// GetBlobRaw fetches a blob's contents from a repo.
+// Unlike GetBlob, it returns the raw bytes rather than the base64-encoded data.
+func (s *GitService) GetBlobRaw(ctx context.Context, owner, repo, sha1 string) ([]byte, error) {
```

Comment:
The endpoint above, and the GitHub API documentation (https://developer.github.com/v3/git/blobs/#get-a-blob) uses `sha` as the name of the parameter (rather than `sha1`). I suspect it's better if we stay consistent.

Reply:
done